services/ffc/ffc-cr/base/helm-repository.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: ffc-cr-helm-repo
+  namespace: flux-config
```

Comment:
Can you please add the labels for the backstage flux plugins to all flux manifests

for team level manifests
**backstage.io/kubernetes-team: ffc-cr** 


for service level, add two labels
backstage.io/kubernetes-id
backstage.io/kubernetes-team